### PR TITLE
Alex Brush: Version 1.101 added

### PR DESCRIPTION
**Pull request by vv-monsalve**
1166af2: [gftools-packager] Alex Brush: Version 1.101 added

* Alex Brush Version 1.101 taken from the upstream repo https://github.com/googlefonts/alex-brush at commit https://github.com/googlefonts/alex-brush/commit/1cc69456e121a16a430ef05477510ab0730a2bfe.

 f3f272a: [gftools-packager] ofl/alexbrush remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by vv-monsalve** [comment]
@davelab6 please confirm this two last issues:

- The only remaining glyph missing is Division Slash, but being a script font I didn't want to stop the PR for it.
🔥 FAIL Missing required codepoints: 0x2215 (DIVISION SLASH) [code: missing-codepoints]

- Should we keep `TypeSETit` as designer for all the Rob's projects coming or should we change it to `Robert E. Leuschke`?
⚠ WARN: METADATA.pb: Designer is listed with the correct name on the Google Fonts catalog of designers?

**Comment by davelab6** [comment]
> The only remaining glyph missing is Division Slash, but being a script font I didn't want to stop the PR for it.
> 🔥 FAIL Missing required codepoints: 0x2215 (DIVISION SLASH) [code: missing-codepoints]

I suggest to make 0x2215 with a component of U+2044 if it exists, and if not, kick it back to Rob or Nhung to create it.

> Should we keep TypeSETit as designer for all the Rob's projects coming or should we change it to Robert E. Leuschke?

`Robert Leuschke` is good

**Comment by davelab6** [comment]
I can see the font being used for poster like situations with fractions. "buy now, ⅞ off" etc

**Comment by vv-monsalve** [comment]
Updated

## Alex Brush: Version 1.101 added

---
 e05570f: [gftools-packager] Alex Brush: Version 1.101 added

* Alex Brush Version 1.101 taken from the upstream repo https://github.com/googlefonts/alex-brush at commit https://github.com/googlefonts/alex-brush/commit/6e3d2e8b04a0926113d9cd8e21a3b4034b295db0.

 53720ac: [gftools-packager] ofl/alexbrush remove METADATA "source".  google/fonts#2587

File: ofl/alexbrush/OFL.txt [hunk 1]
```diff
@@ -1,94 +1,88 @@
...
-THE FONT SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTIES OF
-MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT
-OF COPYRIGHT, PATENT, TRADEMARK, OR OTHER RIGHT. IN NO EVENT SHALL THE
-COPYRIGHT HOLDER BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
-INCLUDING ANY GENERAL, SPECIAL, INDIRECT, INCIDENTAL, OR CONSEQUENTIAL
-DAMAGES, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF THE USE OR INABILITY TO USE THE FONT SOFTWARE OR FROM
-OTHER DEALINGS IN THE FONT SOFTWARE.
+Copyright 2011 The Alex Brush Project Authors (https://github.com/googlefonts/alex-brush)
```
Comment (RosaWagner) [inline ofl/alexbrush/OFL.txt:1]: This part is missing (and so it is blocking the push to sandbox):
```
This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license is copied below, and is also available with a FAQ at:
https://scripts.sil.org/OFL
```

**Pull request merged**